client/src/parsec/login.ts

```diff
@@ -67,18 +72,16 @@ export async function login(device: AvailableDevice, password: string): Promise<
 
   if (!needsMocks()) {
     const clientConfig = getClientConfig();
-    const strategy: DeviceAccessStrategyPassword = {
-      tag: DeviceAccessStrategyTag.Password,
-      password: password,
-      keyFile: device.keyFilePath,
-    };
-    const result = await libparsec.clientStart(clientConfig, parsecEventCallback, strategy);
+    const result = await libparsec.clientStart(clientConfig, parsecEventCallback, accessStrategy);
     if (result.ok) {
       loggedInDevices.push({ handle: result.value, device: device });
     }
     return result;
   } else {
-    if (password === 'P@ssw0rd.' || password === 'AVeryL0ngP@ssw0rd') {
+    if (
+      accessStrategy.tag === DeviceAccessStrategyTag.Password &&
+      ['P@ssw0rd.', 'AVeryL0ngP@ssw0rd'].includes((accessStrategy as DeviceAccessStrategyPassword).password)
+    ) {
```

Comment:
Are we forced to keep this instead of checking true password now?

Reply:
Yes, we still don't have much on web.

Reply:
👌